src/main/java/me/xemor/yaggaskabble/LeaderboardCommand.java

```diff
@@ -31,14 +31,25 @@ public void onSlashCommandInteraction(SlashCommandInteractionEvent event) {
             event.reply("You have entered an invalid winning alignment!").queue();
             return;
         }
-
+        String leaderboardString;
         List<Player> players = yaggaskabble.getPlayers();
-        players.sort(Comparator.comparingDouble(player -> player.getSkillRatingForAlignment(alignment).conservativeRating()));
-        Collections.reverse(players);
-
-        String leaderboardString = players.stream()
+        
+        if (alignment == Alignment.BOTH) {
+            players.sort(Comparator.comparingDouble(player -> player.getSkillRatingForAlignment(Alignment.GOOD).conservativeRating() + player.getSkillRatingForAlignment(Alignment.EVIL).conservativeRating()));
```

Comment:
These two if conditions are the same aside from different implementation of the comparator, you could make it just swap out the comparator.